sharkattack.sh

```diff
@@ -19,6 +22,24 @@ function shark_alias {
     command=$2
     threshold=$3
 
+    SHARK_ATTACK_TYPE='shark'
```

Comment:
Setting these within the `*_alias` functions won't have any effect as `SHARK_ATTACK_TYPE` isn't de-referenced until `shark_swim` is actually called. You probably want to scrap the `SHARK_ATTACK_TYPE` from this file and make a `cat_meow` function analogous to `shark_swim` to complement `cat_alias`.